pkgs/development/python-modules/pyTwitchAPI/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, aiohttp
+, python-dateutil
+, requests
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "pyTwitchAPI";
+  version = "2.3.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Teekeks";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ax3FHyyyRfXSWKsoUi8ao5TL2alo0bQP+lWiDaPjf34=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    python-dateutil
+    requests
+    websockets
+  ];
+
+  doChecks = false;
```

Comment:
Please add a comment why we don't run the tests.

Reply:
Changes implemented